contrib/netbsd-tests/usr.bin/grep/t_grep.sh

```diff
@@ -426,6 +426,13 @@ color_body()
 
 	atf_check -o file:"$(atf_get_srcdir)/d_color_c.out" \
 	    grep --color=always -f grepfile "$(atf_get_srcdir)/d_color_b.in"
+
```

Comment:
This empty line is not present in the NetBSD test suite. While aesthetically pleasing, it worsens the diffs when importing the changes from NetBSD.

Reply:
Right. Fixed.